prow/config/config.go

```diff
@@ -2199,7 +2212,6 @@ func (c Config) validatePeriodics(periodics []Periodic) error {
 // ValidateJobConfig validates if all the jobspecs/presets are valid
 // if you are mutating the jobs, please add it to finalizeJobConfig above.
 func (c *Config) ValidateJobConfig() error {
-
```

Comment:
Let's not introduce diffs like these that are not related to the code that's being changed.

Reply:
Removed 👍